app/src/main/res/layout/component_awesomebar.xml

```diff
@@ -14,6 +14,7 @@
     android:requiresFadingEdge="vertical"
     app:layout_constraintTop_toTopOf="@id/awesomeBar_barrier"
     app:layout_constraintEnd_toEndOf="parent"
+    app:fontFamily="@font/inter_regular"
```

Comment:
Is there a reason we're not just applying a style to this 🤔 does AC not support that?